sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -45,7 +47,8 @@ private[sql] object JDBCRelation {
    * incorrect values may cause the partitioning to be poor, but no data
    * will fail to be represented.
    */
-  def columnPartition(partitioning: JDBCPartitioningInfo): Array[Partition] = {
+  def columnPartition(partitioning: JDBCPartitioningInfo,
+    schema: StructType, url: String): Array[Partition] = {
```

Comment:
can you add some documentation to this function to explain the parameters?

just do them with @param